src/util/fields.py

```diff
@@ -1149,17 +1154,23 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
     col_prefix = ""
     if not column_exists(cr, "ir_act_server", "condition"):
         col_prefix = "--"  # sql comment the line
-
+    standard_modules = modules.get_modules()
```

Comment:
```suggestion
    standard_modules = modules.get_modules() | {"__upgrade__"}
```

Reply:
@KangOl Thank you for your valuable time. The suggested changes have been implemented.